lib/ansible/modules/cloud/vmware/vmware_dvs_host.py

```diff
@@ -49,6 +51,14 @@
         choices: [ present, absent ]
         required: True
         default: 'present'
+    vendor_specific_config:
```

Comment:
Code lgtm, could you please add an example for this new option ? This will be easy for end user to understand how this parameter is being used in this module.

Reply:
Thanks, I've added an example using the new field.